install.js

```diff
@@ -17,6 +17,10 @@ function sanitizeURL(inputURL) {
 // So we are 100% sure that the saucectl binary will be available for the next
 // execution.
 async function install() {
+  if (!process.env.FORCE_INSTALL_SAUCECTL && process.env.SAUCE_VM) {
```

Comment:
(Optional) Just a personal preference in this case, as this order follows the prose logic better: "if it's on a Sauce VM, skip installation, unless installation is forced anyway".
```suggestion
  if (process.env.SAUCE_VM && process.env.FORCE_INSTALL_SAUCECTL) {
```

Currently it reads: "if forced installation is disabled and it's on a sauce vm, skip installation" -> my brain takes longer to parse the intent.

Reply:
Is it
```
  if (process.env.SAUCE_VM && !process.env.FORCE_INSTALL_SAUCECTL) {
```